.github/workflows/lbox-develop.yml

```diff
@@ -2,9 +2,9 @@ name: LBox Develop
 
 on:
   push:
-    branches: [develop]
+    branches: [develop, v6]
```

Comment:
Note that we need to remove v6 from the workflows before merging to develop. These are here temporary for tests to run